scripts/set_xtensa_params.sh

```diff
@@ -118,6 +118,13 @@ case "$platform" in
 	HOST="xtensa-acp_6_3-elf"
 	TOOLCHAIN_VER="RI-2021.6-linux"
 	;;
+	acp_7_0)
+ 	PLATFORM="acp_7_0"
+ 	ARCH="xtensa"
```

Comment:
Are you sure `ARCH=` is required here? A lot of platforms do not have it. Can you try without it?